<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageTcpCLient</class>
 <widget class="QWidget" name="ManageTcpCLient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>343</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Secure Socket Client</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>10</y>
     <width>784</width>
     <height>581</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="hostNameLabel">
        <property name="text">
         <string>Host name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="hostNameEdit">
        <property name="text">
         <string>www.zhangxianyi4work.work</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="portLabel">
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="portBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>9999</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Active session</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="connectButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Connect to host</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="search_layout">
      <item>
       <widget class="QLineEdit" name="root_path_lineEdit">
        <property name="text">
         <string>G:\QT-project</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_search">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>search</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="client_lab">
        <property name="text">
         <string>客户端文件列表：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="sessionout_lab">
        <property name="text">
         <string>文本信息显示：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="server_lab">
        <property name="text">
         <string>服务器文件列表：</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="show_layout">
      <item>
       <widget class="QListView" name="client_listView"/>
      </item>
      <item>
       <widget class="QTextEdit" name="sessionOutput">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Sans Serif';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="server_listView">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="curr_dir_lab">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>curr_dir:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="btn_layout">
      <item>
       <widget class="QPushButton" name="btn_upload">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>upload</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_download">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>load</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_preview">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>preview</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QLabel" name="sessionInputLabel">
        <property name="text">
         <string>Input:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="sessionInput">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sendButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="text">
         <string>&amp;Send</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>hostNameEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>connectButton</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sessionInput</sender>
   <signal>returnPressed()</signal>
   <receiver>sendButton</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
